domain/src/main/java/com/teamagam/gimelgimel/domain/alerts/InformNewAlertsInteractor.java

```diff
@@ -3,37 +3,57 @@
 import com.google.auto.factory.AutoFactory;
 import com.google.auto.factory.Provided;
 import com.teamagam.gimelgimel.domain.alerts.entity.Alert;
+import com.teamagam.gimelgimel.domain.alerts.entity.AlertPresentation;
 import com.teamagam.gimelgimel.domain.alerts.repository.AlertsRepository;
 import com.teamagam.gimelgimel.domain.alerts.repository.InformedAlertsRepository;
 import com.teamagam.gimelgimel.domain.base.executor.PostExecutionThread;
 import com.teamagam.gimelgimel.domain.base.executor.ThreadExecutor;
 import com.teamagam.gimelgimel.domain.base.interactors.BaseSingleDisplayInteractor;
 import com.teamagam.gimelgimel.domain.base.interactors.DisplaySubscriptionRequest;
+import com.teamagam.gimelgimel.domain.messages.entity.ChatMessage;
+import com.teamagam.gimelgimel.domain.messages.entity.features.TextFeature;
+import com.teamagam.gimelgimel.domain.messages.repository.MessagesRepository;
+import com.teamagam.gimelgimel.domain.messages.repository.ObjectMessageMapper;
 import java.util.Date;
+import javax.inject.Named;
 
 @AutoFactory
 public class InformNewAlertsInteractor extends BaseSingleDisplayInteractor {
 
   private final AlertsRepository mAlertsRepository;
   private final InformedAlertsRepository mInformedAlertsRepository;
   private final Displayer mDisplayer;
+  private ObjectMessageMapper mObjectMessageMapper;
+  private MessagesRepository mMessagesRepository;
 
   InformNewAlertsInteractor(@Provided ThreadExecutor threadExecutor,
       @Provided PostExecutionThread postExecutionThread,
       @Provided AlertsRepository alertsRepository,
       @Provided InformedAlertsRepository informedAlertsRepository,
-      Displayer displayer) {
+      Displayer displayer,
```

Comment:
Re-organize parameters order so that the `@Provided` parameters are before the others

Reply:
done